### PR TITLE
Update to June 2020 3.1.105 release.

### PR DESCRIPTION
**Pull request by crummel**

### CODE REVIEW

**Comment by crummel** [comment]
https://github.com/dotnet/source-build/pull/1616/files#diff-44f2a3d3b6a84d3986f2778256ae765d is what I ended up doing for https://github.com/dotnet/source-build/issues/1594 - remove the need to patch authenticated feeds out.

File: patches/aspnetcore/0016-remove-temp-feeds.patch [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+-    <add key="darc-int-dotnet-efcore-5f01ecc" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-efcore-5f01ecc8/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-core-setup-65cd789" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-core-setup-65cd7897/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-corefx-620cea9" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-620cea9c/nuget/v3/index.json" />
 -    <add key="darc-int-dotnet-corefx-059a4a1" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-059a4a19/nuget/v3/index.json" />
--    <add key="darc-int-dotnet-aspnetcore-tooling-a8242d7" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-aspnetcore-tooling-a8242d79/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-aspnetcore-tooling-0e01666" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-aspnetcore-tooling-0e016665/nuget/v3/index.json" />
 -    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
```
Comment (dagood) [inline patches/aspnetcore/0016-remove-temp-feeds.patch:26]: Is it possible to remove this patch entirely too, or am I missing something unusual here?

File: tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs [hunk 2]
```diff
@@ -22,6 +22,8 @@ public class RemoveInternetSourcesFromNuGetConfig : Task
         [Required]
         public string NuGetConfigFile { get; set; }
 
+        public bool OfflineBuild { get; set; }
```
Comment (dagood) [inline tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs:25]: The task comment looks like it's describing something else--remove it? Maybe add a comment on this property to explain what it's doing in particular?

File: tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs [hunk 3]
```diff
@@ -31,7 +33,14 @@ public override bool Execute()
...
                 if (e.Name == "add")
                 {
-                    return !(e.Attribute("value").Value.StartsWith("http://", StringComparison.OrdinalIgnoreCase) || e.Attribute("value").Value.StartsWith("https://", StringComparison.OrdinalIgnoreCase));
+                    if (OfflineBuild)
+                    {
+                        return !(e.Attribute("value").Value.StartsWith("http://", StringComparison.OrdinalIgnoreCase) || e.Attribute("value").Value.StartsWith("https://", StringComparison.OrdinalIgnoreCase));
+                    }
+                    else
+                    {
+                        return !(e.Attribute("value").Value.StartsWith("https://pkgs.dev.azure.com/dnceng/_packaging", StringComparison.OrdinalIgnoreCase));
```
Comment (dagood) [inline tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs:42]: Factor out `e.Attribute("value").Value.` to make both branches easier to read?

File: patches/aspnetcore/0016-remove-temp-feeds.patch (hunk 1 above)
Comment (crummel) [reply patches/aspnetcore/0016-remove-temp-feeds.patch:26]: Yes, think I just missed that one in the merge, thanks.  Removed.

File: tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs (hunk 2 above)
Comment (crummel) [reply tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs:25]: Fixed the task comment and added a comment here.

File: tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs (hunk 3 above)
Comment (crummel) [reply tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/RemoveInternetSourcesFromNuGetConfig.cs:42]: Good idea, done.

**Pull request merged**